Wbooks/Detail/Book Detail/BookDetailController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  BookDetailController.swift
+//  Wbooks
+//
+//  Created by Juan Martín Gordo on 21/04/2021.
+//
+
+import UIKit
+
+final class BookDetailController: UIViewController {
+
+    // MARK: Properties
+    private lazy var bookDetailView: BookDetailView = BookDetailView()
+    let viewModel: BookDetailViewModel
+    let errorTitle = NSLocalizedString("ALERT_BOX.TITLE", comment: "Title for the error alert box")
+    let rentErrorMessage = NSLocalizedString("ALERT_BOX.ERROR_MESSAGE", comment: "Message detailing an error in the alert box")
+    let errorMessage = NSLocalizedString("ALERT_BOX.BOOK_UNAVAILABLE_MESSAGE", comment: "Message detailing an error in the alert box")
+    let rentSuccessMessage = NSLocalizedString("ALERT_BOX.BOOK_RENTED", comment: "Message detailing an error in the alert box")
+    let errorDismiss = NSLocalizedString("ALERT_BOX.BUTTON", comment: "Text for the dismiss button on the alert box")
+
+    // MARK: Lifecycle methods
+    override func loadView() {
+        view = bookDetailView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupRentButton()
+        setupAddToWishlistButton()
+    }
+    
+    func setupView() {
+        bookDetailView.addToWishlistButton.setSecondaryStyle()
+        if viewModel.status == "Available" {
```

Comment:
I think it would be a good idea to add another method to the view model that returns a Boolean:
```Swift
func viewModel.isAvailable() -> Bool {
   return book.status == "Available"
}
```
In this way we avoid comparing strings multiple times.